cdk/lib/__snapshots__/security-hq.test.ts.snap

```diff
@@ -1079,42 +1092,6 @@ dpkg -i /tmp/installer.deb",
       },
       "Type": "AWS::IAM::Policy",
     },
-    "SSMRunCommandPolicy244E1613": {
```

Comment:
This change originates from https://github.com/guardian/cdk/pull/1694 and GuCDK v49.0.2.